Sources/PodToBUILD/ObjcLibrary.swift

```diff
@@ -89,26 +89,29 @@ public struct ConfigSetting: BazelTarget {
     }
 }
 
-// https://bazel.build/versions/master/docs/be/objective-c.html#objc_framework
-public struct ObjcFramework: BazelTarget {
+// https://github.com/bazelbuild/rules_apple/blob/818e795208ae3ca1cf1501205549d46e6bc88d73/doc/rules-general.md#apple_static_framework_import
+public struct AppleStaticFrameworkImport: BazelTarget {
```

Comment:
We will need to add a load statement into files that use this target type.

I've made this a lot easier to do in https://github.com/pinterest/PodToBUILD/pull/68 so lets merge that first and then update this PR